index.html

```diff
@@ -0,0 +1,259 @@
+		<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+		<meta name="language" content="English">
+		<meta name="image" content="https://www.impactlabs.io/assets/impact_labs.jpg">
+
+		<meta itemprop="name" content="Impact Labs: Tech for Social Good">
+		<meta itemprop="description" content="Impact Labs is an organization inspiring and empowering ambitious computer scientists to build a better world.">
+		<meta itemprop="image" content="https://www.impactlabs.io/assets/impact_labs.jpg">
+
+		<meta name="twitter:card" content="summary">
+		<meta name="twitter:title" content="Impact Labs: Tech for Social Good">
+		<meta name="twitter:description" content="Impact Labs is an organization inspiring and empowering ambitious computer scientists to build a better world.">
+		<meta name="twitter:image:src" content="https://www.impactlabs.io/assets/impact_labs.jpg">
+
+		<meta property="og:title" content="Impact Labs: Tech for Social Good">
+		<meta property="og:description" content="Impact Labs is an organization inspiring and empowering ambitious computer scientists to build a better world.">
+		<meta property="og:image" content="https://www.impactlabs.io/assets/impact_labs.jpg">
+		<meta property="og:url" content="https://www.impactlabs.io/">
+		<meta property="og:site_name" content="Impact Labs">
+		<meta property="og:type" content="website">
+
+		<title>Impact Labs</title>
+
+		<link href="./css/main.css" rel="stylesheet">
+		<link href="https://unpkg.com/aos@2.3.1/dist/aos.css" rel="stylesheet">
+
+		<!-- Global site tag (gtag.js) - Google Analytics -->
+		<script async src="https://www.googletagmanager.com/gtag/js?id=UA-137998785-1"></script>
+		<script>
+			window.dataLayer = window.dataLayer || [];
+			function gtag(){dataLayer.push(arguments);}
+			gtag('js', new Date());
+
+			gtag('config', 'UA-137998785-1');
+		</script>
+	</head>
+
+	<body id="page-top">
+
+		<nav class="navbar navbar-expand-md navbar-dark bg-primary fixed-top" id="mainNav">
+		<div class="container-fluid">
+			<a class="navbar-brand" href="./">
+			<img alt="Impact Logo" src="./assets/icons/impact_labs_white.svg"/>
+			</a>
+			<button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+			<span class="navbar-toggler-icon"></span>
+			</button>
+			<div class="collapse navbar-collapse" id="navbarResponsive">
+			<ul class="navbar-nav ml-auto">
+				<li class="nav-item active">
+				<a class="nav-link" href="./">Home</a>
+				</li>
+				<li class="nav-item">
+				<a class="nav-link" href="./fellowship">Fellowship</a>
+				</li>
+				<li class="nav-item">
+				<a class="nav-link" href="./summit">Summit</a>
+				</li>
+				<li class="nav-item">
+				<a class="nav-link" href="./coalition">Coalition</a>
+				</li>
+				<li class="nav-item">
+				<a class="nav-link" href="./team">Team</a>
+				</li>
+				<li class="nav-item">
```

Comment:
Not entirely sure how this relates to the changes in `data/nav.yaml`, should these tabs be updated to reflect the changes (About Us instead of Team and removing the blog)?

Reply:
Great catch! Will fix that now